django/contrib/admin/options.py

```diff
@@ -962,12 +962,22 @@ def log_deletion(self, request, obj, object_repr):
             action_flag=DELETION,
         )
 
-    @display(description=mark_safe('<input type="checkbox" id="action-toggle">'))
+    @display(
+        description=mark_safe(
+            '<input type="checkbox" id="action-toggle" '
+            'aria-label="Select all objects on this page for an action">'
+        )
+    )
     def action_checkbox(self, obj):
         """
         A list_display column containing a checkbox widget.
         """
-        return helpers.checkbox.render(helpers.ACTION_CHECKBOX_NAME, str(obj.pk))
+        attrs = {
+            "class": "action-select",
+            "aria-label": "Select this object for an action - %s" % obj,
```

Comment:
We should add both labels to translations.

Reply:
Ok

Reply:
When I wrapped the strings passed in [`@display` with the `gettext_lazy function`](https://github.com/django/django/pull/16515/files#diff-e77dfa6935de3fac94356ffd1029a679f8aa23f2979fe9b31aea7f93e1dccad3R969), the test routine started to break with the following message:

```bash
[...]
File "/home/durval/gsoc/django/django/utils/translation/trans_real.py", line 212, in _add_installed_apps_translations
    raise AppRegistryNotReady(
django.core.exceptions.AppRegistryNotReady: The translation infrastructure cannot be initialized before the apps registry is ready. Check that you don't make non-lazy gettext calls at import time.
```

I don't know how to solve this problem. I looked for some examples of using the `gettext_lazy` function in decorators and didn't find any. Do you have any suggestion?

Reply:
It's not related with decorators, you tried to add `gettext_lazy()` to `str()` so it's not lazy anymore.

Reply:
Oh... Makes sense 😅 
[I made the changes](https://github.com/django/django/blob/41ac7c8e532fe16a1e21be9f8928d89411ad8724/django/contrib/admin/options.py#L967)